src/main/java/controller/BetHandler.java

```diff
@@ -0,0 +1,33 @@
+package controller;
+
+import dao.BetDao;
+import dao.impl.BetDaoImpl;
+import java.util.Scanner;
+import model.Bet;
+
+public class BetHandler implements ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+
+    @Override
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (ArrayIndexOutOfBoundsException | NumberFormatException e) {
+                System.out.println("Please,enter correct data");
```

Comment:
should we add our bet if these exceptions had been thrown?

Reply:
Sry,but i didn`t understand what do you mean